### PR TITLE
CHIRPS

### PR DESCRIPTION
**Pull request by norlandrhagen**

### CODE REVIEW

**Comment by norlandrhagen** [comment]
@andersy005 it seems like the CI validation checks are stuck?

**Comment by andersy005** [comment]
> @andersy005 it seems like the CI validation checks are stuck. Any ideas?

this is likely due to a feature in GitHub Actions designed to prevent recursive workflow runs. will look into it.

File: catalog/input.yaml [hunk 1]
```diff
@@ -18,3 +18,4 @@ feedstocks:
   - https://github.com/leap-stc/WOA-1-degree-monthly-pangeo-forge-osn-feedstock/blob/main/feedstock/catalog.yaml
   - https://github.com/leap-stc/LLC4320_12h_3D-regions_feedstock/blob/main/feedstock/catalog.yaml
   - https://github.com/leap-stc/eNATL_feedstock/blob/main/feedstock/catalog.yaml
+  - https://github.com/leap-stc/chirps_feedstock/blob/main/feedstock/meta.yaml
```
Comment (andersy005) [inline catalog/input.yaml:21]: this needs to point to the `catalog.yaml` file instead

```suggestion
  - https://github.com/leap-stc/chirps_feedstock/blob/main/feedstock/catalog.yaml
```

**Review by andersy005 — APPROVED** [review]

**Comment by andersy005** [comment]
@norlandrhagen, is this ready? the catalog preview looks good

**Comment by andersy005** [comment]
> > @andersy005 it seems like the CI validation checks are stuck. Any ideas?
> 
> this is likely due to a feature in GitHub Actions designed to prevent recursive workflow runs. will look into it.

this issue has been fixed

**Pull request merged**